### DISCUSSION TITLE
Odd Boundary Condition Behavior with Wind Modeling

### DISCUSSION THREAD

**Discussion by sebastianmezaurotadze** [firemodels/fds#13345] [Q&A]
I want to model wind gusts in a simulation, where the wind is defined on the &WIND line,  starting at 0 at the beginning of the simulation and then increases at a later time stamp. However, I notice that the open boundary conditions on the wind simulation somewhat act as walls when wind speed is set to 0 m/s. Is there a way to make the open boundary conditions act as open boundaries instead of constraining the flow when wind speed is set to 0 m/s?

**No Wind**
https://github.com/user-attachments/assets/732109ca-18bc-43d8-aa73-3333152eb94b

**Wind = 0 m/s** 
https://github.com/user-attachments/assets/dd631594-2158-4d5c-8ccd-7e0970457995

See related ticket on: https://github.com/firemodels/fds/discussions/13329

**Comment by mcgratta** [comment]
`PRESSIRE_GRADIENT_FORCE`. Search the User's Guide. This parameter allows you to impose a pressure gradient over the entire domain that is based on the way the wind actually works. It does not, however, rigorously force the wind to be exactly what you want. You have to experiment a bit to get the surface friction and forcing to balance at your desired speed.

**Reply by sebastianmezaurotadze** [reply]
I tried this and it seems to work well! I overall also see more stable behavior with this method. Anytime fire reaches the edge of the domain on with the velocity defined on the &WIND line it tends to fail due to numerical instability. However, this problem seems to go away when no wind is specified or if pressure wind is used instead. Do you know why this might be?

**Reply by ericvmueller** [reply]
Which version are you using? Something like this was addressed with #12598 between versions 6.9 and 6.9.1

**Reply by matt-s-distefano** [reply]
Sorry, to clarify, do you mean that the fix between 6.9.0 and 6.9.1 addressed both numerical instability **and** the behavior where at zero velocity the open boundary conditions essentially become a wall boundary condition? Or is the version adjustment focused **only** on the numerical instability? I'm reading through the https://github.com/firemodels/fds/issues/12574 fix and it seems geared only towards the numerical instability, but I wanted to double check to see if I was missing anything, thanks!
